storybookjs/storybook
# Build: Improve e2e script

## What I did

 - Improve CLI with prompts
 - Clean some duplicated code (use function from SB CLI directly)
 - Reduce log and improve e2e output with a sum-up of the passing/failing tests

## How to test

- Locally: ` yarn local-registry --port 6000 --open --publish` and `yarn test:e2e-framework --clean --use-local-sb-cli cra`
- On CI: the e2e steps should be way more understandable
